style.css

```diff
@@ -613,14 +613,19 @@ a.current{
     text-align:center;
 }
 
+.products p.subheading{
+    text-align:center;
+    font-size:20px;
+}
+
 .container{
     margin-left:1.5%;
     height:auto;
 }
 
 .product-box{
     float:left;
-    height:auto;
+    height:369.64px;

```

Comment:
I had to add an exact box size so that long product names wouldn't stretch the box. This isn't ideal, but for now it keeps the boxes in the exact size that I want.